spec/workers/feeds/import_articles_worker_spec.rb

```diff
@@ -20,7 +20,7 @@
       allow(Feeds::Import).to receive(:call)
 
       Timecop.freeze(Time.current) do
-        worker.perform(1.minute.ago)
+        worker.perform([], 1.minute.ago)
```

Comment:
We only ever use "earlier than" in this test, asserting that the option exists and works.